superset/connectors/druid/views.py

```diff
@@ -176,6 +177,8 @@ class DruidClusterModelView(SupersetModelView, DeleteMixin, YamlExportMixin):  #
         'cluster_name': _('Cluster'),
         'broker_host': _('Broker Host'),
         'broker_port': _('Broker Port'),
+        'broker_user': _('Broker Username'),
```

Comment:
Sorry one more last thing, can we make it clear in the `description_columns` right under here that these 2 columns are optional and guidance (maybe a link to the druid docs) on how to set this up on the server side.